handler/handler.go

```diff
@@ -719,17 +777,26 @@ func score(category yahtzee.Category, dices []int) (int, error) {
 			s = 40
 		}
 	case yahtzee.Yahtzee:
-		same := true
-		for i := 0; i < len(dices)-1; i++ {
-			same = same && dices[i] == dices[i+1]
-		}
-
-		if same {
-			s = 50
+		for i := 1; i < 6; i++ {
```

Comment:
because there are 6 sides of a dice.
This is a very basic algorithm, I just count for every value how many dices are there, if one value has 5 dices, then we have a yahtzee